.cicdtemplate/.github/workflows/deploy_production_to_firebase_app_distribution.yml

```diff
@@ -0,0 +1,68 @@
+name: Deploy staging and production to Firebase App Distribution
+
+on:
+  # Trigger the workflow on push action in main branch.
+  # So it will trigger when the PR of the feature branch was merged.
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy_staging_and_production_to_firebase_app_distribution:
+    name: Deploy staging and production to Firebase App Distribution
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+
+      - name: Set up timezone
+        uses: zcong1993/setup-timezone@master
+        with:
+          timezone: Asia/Bangkok
+
```

Comment:
I'm sure deploy workflow does not need to `Set up timezone` step 🙏  Same for staging workflow